.circleci/src/jobs/@root-jobs.yml

```diff
@@ -0,0 +1,29 @@
+# Publish npm packages
+root-publish-npm-packages:
+  working_directory: ~/audius-client
+  docker:
+    - image: circleci/node:14.18
+  steps:
+    - checkout
+    - run:
+        name: npm Auth
+        command: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > .npmrc
+    - run:
+        name: Publish packages
+        command: npx lerna publish from-git --yes
```

Comment:
any way we can depend on not npx to do this? like from node modules or something like that?

Reply:
Oh yeah there should be an `npm i` step first so we aren't making a network call with npx. But I think it's still ok to use npx once the packages are installed bc it will just use the local one

Reply:
cool, sgtm!